--- oreon-1.2.3-RC2/oreon_src/install/setup.php~	2006-03-14 21:31:09.000000000 +0200
+++ oreon-1.2.3-RC2/oreon_src/install/setup.php	2006-03-14 21:33:06.000000000 +0200
@@ -150,19 +150,19 @@
 		  </tr>
 		  <tr>
             <td style="padding-left:50px ">Nagios configuration directory</td>
-			<td><input name="nagios_conf" type="text" value="/usr/local/nagios/etc/" size="40"></td>
+			<td><input name="nagios_conf" type="text" value="/etc/nagios" size="40"></td>
 		  </tr>
 		  <tr>
             <td style="padding-left:50px ">Nagios plugins</td>
-			<td><input name="nagios_plugins" type="text" value="/usr/local/nagios/libexec/" size="40"></td>
+			<td><input name="nagios_plugins" type="text" value="/usr/lib/nagios/plugins" size="40"></td>
 		  </tr>
 		  <tr>
             <td style="padding-left:50px ">RRDTool directory install</td>
-			<td><input name="rrdtool_dir" type="text" value="/usr/local/rrdtool-1.0.49/bin/" size="40"></td>
+			<td><input name="rrdtool_dir" type="text" value="/usr/bin" size="40"></td>
 		  </tr>
 		  <tr>
             <td style="padding-left:50px ">SNMP binary path</td>
-			<td><input name="snmp_dir" type="text" value="/usr/local/bin" size="40"></td>
+			<td><input name="snmp_dir" type="text" value="/usr/bin" size="40"></td>
 		  </tr>
 		</table>
 		<?
@@ -222,7 +222,7 @@
 		  <tr>
             <td><b>Writable Oreon Configuration File (oreon.conf.php)</b></td>
             <td align="right"><?
-				if(is_writable('..')){
+				if(is_writable('../oreon.conf.php')){
                   	echo '<b><span class="go">OK</font></b>';
 				} else {
                   	echo '<b><span class="stop">Warning: Not Writeable</font></b>';
@@ -505,9 +506,9 @@
 			</tr>
 			<?
 			if (!$_SESSION["pwdroot"])
-				$res = connexion('root', '', $_SESSION["dbLocation"]) or die ("Can't connect to Mysql Server : ".mysql_error());
+				$res = connexion('mysql', '', $_SESSION["dbLocation"]) or die ("Can't connect to Mysql Server : ".mysql_error());
 			else
-				$res = connexion('root', $_SESSION["pwdroot"], $_SESSION["dbLocation"]) or die ("Can't connect to Mysql Server : ".mysql_error()); //$_SESSION["pwdroot"]
+				$res = connexion('mysql', $_SESSION["pwdroot"], $_SESSION["dbLocation"]) or die ("Can't connect to Mysql Server : ".mysql_error()); //$_SESSION["pwdroot"]
 			$requete = "CREATE DATABASE ". $_SESSION["nameOreonDB"] . ";";
 			if ($DEBUG) print $requete . "<br>";
 			mysql_query($requete, $res);
